source/download/alternate_downloads.md

```diff
@@ -1,32 +1,35 @@
 ---
 title: Download
-authors: gshereme, sandrobonazzola
+authors: gshereme,sandrobonazzola
+page_classes: download
 ---
 
 # Alternate Installation Methods
 
-For production installations, we recommend installing oVirt using the graphical Cockpit installer in the
+<img class="screenshot" src="download_1.png">
+
+For production installations, we recommend installing oVirt in the
 [Self-Hosted Engine configuration](/download). In this configuration, oVirt Engine and a Host are installed
 together with the Engine running as a Virtual Machine on that Host. This configuration is
 preferred because the Engine Virtual Machine will be highly available (once a second Host is added).
 
 However, if you prefer to run oVirt Engine standalone on physical hardware or another virtualization provider, you can install oVirt Engine
 and Nodes / Hosts separately.
```

Comment:
@sandrobonazzola You should choose one word (hosts or nodes) and stick with it.

Also whichever you choose, it shouldn't be capitalized.

Self-Host Engine should only be capitalized if it appears at the beginning of a sentence, and then it should be "Self-hosted engine"

Reply:
I guess we needs to sit and review the wording conventions across the whole website. Can be done in separate patch